code/modules/mob/living/silicon/ai/say.dm

```diff
@@ -135,6 +135,22 @@ var/const/VOX_PATH = "sound/vox_fem/"
 	for(var/word in words)
 		play_vox_word(word, src.z, null)
 
+	ai_voice_announcement_to_text(words, src)
+
+
+/proc/ai_voice_announcement_to_text(words, ai)
+	var/mob/living/silicon/ai/AI = ai
+	var/words_string = jointext(words, " ")

```

Comment:
The words here can be any case possible. Just to keep that in mind.

Reply:
It should be fine since AI voice announcement words are restricted to either Title Case or UPPERCASE